src/scripts/h5p-cornell-content.js

```diff
@@ -366,12 +366,8 @@ export default class CornellContent {
      * given by the user and there's no order stored previously - preventing
      * to show up that restart button without the need to.
      */
-    if (!this.getAnswerGiven()) {
-      return;
-    }
-
     return {
-      dateString: this.previousState.dateString,
+      dateString: this.getAnswerGiven() ? this.previousState.dateString : undefined,
```

Comment:
`this.previousState.dateString` will always be set when the content class is instantiated. It's being sanitized in https://github.com/otacke/h5p-cornell/blob/master/src/scripts/h5p-cornell-content.js#L27-L35

Reply:
yes ?

Reply:
Oh, sorry, I am too dumb to read code right now ... Had to do business foo all morning.

Reply:
ah yes 😌